@here/harp-examples/webpack.config.js

```diff
@@ -195,9 +198,23 @@ const assets = [
         toType: "dir"
     },
     require.resolve("three/build/three.min.js"),
-    resolveOptional("@here/harp.gl/dist/harp.js", "bundle examples require `yarn build-bundle`"),
-    resolveOptional("@here/harp.gl/dist/harp-decoders.js")
-].filter(asset => asset); // ignore stuff that is not found
+    {
+        from: resolveOptional(`@here/harp.gl/dist/harp${harpBundleSuffix}.js`, "bundle examples require `yarn build-bundle`"),
+        to: "harp.js"
+    },
+    {
+        from: resolveOptional(`@here/harp.gl/dist/harp-decoders${harpBundleSuffix}.js`),
+        to: "harp-decoders.js"
+    }
+].filter(asset => {
+    if (asset === undefined || asset === null) {
+        return false;
+    } else if (typeof asset === "string") {
+        return true;
+    } else if (typeof asset === "object") {
+        return asset.from;
+    }
+}); // ignore stuff that is not found
```

Comment:
maybe it would be better:
```
.filter(asset => asset !== undefined && asset !== null && asset.from !== undefined);
```

Reply:
I don't think this will work. The array has a mix of strings and { from: string, to: string } objects. Plain strings should also pass the test. So the from property is only required if it is not a string.